Fix due to aboot machine  tests

Yarboa opened the pull request:
Changing to containers.kill instead of containers.stop python api It reduce failures during cleanup.
containers killed instead of stop+wait option exist in API.

Updated test timeouts to complete success in slow OS.

Yarboa committed:
Fix due to aboot machine  tests

Changing to containers.kill instead of containers.stop python api
It reduce failures during cleanup.
containers killed instead of stop+wait option exist in API.

Updated test timeouts to complete success in slow OS.

Signed-off-by: Yariv Rachmani <yrachman@redhat.com>

engelmi commented on tests/bluechi_test/container.py:
```diff
@@ -60,7 +60,7 @@ def gather_journal_logs(self, data_dir: str) -> None:
     def cleanup(self):
         if self.container.status == 'running':
-            self.container.stop()
+            self.container.kill()
```
I don't think `kill` is a good approach here as it doesn't allow for a graceful shutdown, thus creating zombie containers. These can not be removed and cleaned up properly - leading the subsequent tests to fail (if they use the same name for a container).

Yarboa replied:
From the tests, I ran locally,
tests timeouts and failures were reduced significantly
need to check

But yes, I think we need to treat containers which were not removed and not running also

Yarboa replied:
OK just add the time to wait before kill in podman stop api

engelmi replied:
Looks good!

engelmi reviewed (commented):
Increasing the timeouts makes sense. I'd keep the `stop` though.

Yarboa committed:
Changing Containers.kill -> back to Containers.stop(**)

It seens that the default wait adds 10 sec to each
container stop command,

from podman stop --helo
 -t, --time uint
 Seconds to wait for stop before killing the container (default 10)

Signed-off-by: Yariv Rachmani <yrachman@redhat.com>

engelmi reviewed (approved):
LGTM

Yarboa commented on tests/tests/tier0/proxy-service-stop-target/test_proxy_service_stop_target.py:
```diff
@@ -53,7 +53,7 @@ def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer
 
 
-@pytest.mark.timeout(10)
+@pytest.mark.timeout(20)
```
```suggestion
@pytest.mark.timeout(40)
```

Yarboa commented on tests/tests/tier0/proxy-service-stop-requesting/test_proxy_service_stop_requesting.py:
```diff
@@ -53,7 +53,7 @@ def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer
 
 
-@pytest.mark.timeout(10)
+@pytest.mark.timeout(20)
```
```suggestion
@pytest.mark.timeout(40)
```

Yarboa commented on tests/tests/tier0/proxy-service-stop-requesting-with-unneeded/test_proxy_service_stop_requesting_with_unneeded.py:
```diff
@@ -53,7 +53,7 @@ def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer
 
 
-@pytest.mark.timeout(10)
+@pytest.mark.timeout(20)
```
```suggestion
@pytest.mark.timeout(40)
```

Yarboa commented on tests/tests/tier0/proxy-service-stop-bluechi-dep/test_proxy_service_stop_bluechi_dep.py:
```diff
@@ -53,7 +53,7 @@ def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer
 
 
-@pytest.mark.timeout(10)
+@pytest.mark.timeout(20)
```
```suggestion
@pytest.mark.timeout(40)
```

Yarboa commented on tests/tests/tier0/proxy-service-start/test_proxy_service_start.py:
```diff
@@ -41,7 +41,7 @@ def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer
 
 
-@pytest.mark.timeout(10)
+@pytest.mark.timeout(20)
```
```suggestion
@pytest.mark.timeout(40)
```

Yarboa commented on tests/tests/tier0/proxy-service-multiple-services-one-node/test_proxy_service_multiple_services_one_node.py:
```diff
@@ -57,7 +57,7 @@ def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer
 
 
-@pytest.mark.timeout(15)
+@pytest.mark.timeout(20)
```
```suggestion
@pytest.mark.timeout(40)
```

Yarboa commented on tests/tests/tier0/proxy-service-multiple-services-multiple-nodes/test_proxy_service_multiple_services_multiple_nodes.py:
```diff
@@ -58,7 +58,7 @@ def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer
 
 
-@pytest.mark.timeout(15)
+@pytest.mark.timeout(25)
```
```suggestion
@pytest.mark.timeout(40)
```

Yarboa commented on tests/tests/tier0/proxy-service-fails-on-typo-in-file/test_proxy_service_fails_on_typo_in_file.py:
```diff
@@ -41,7 +41,7 @@ def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer
 
 
-@pytest.mark.timeout(10)
+@pytest.mark.timeout(20)
```
```suggestion
@pytest.mark.timeout(40)
```

Yarboa commented on tests/tests/tier0/proxy-service-fails-on-non-existent-service/test_proxy_service_fails_on_non_existent_service.py:
```diff
@@ -35,7 +35,7 @@ def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer
 
 
-@pytest.mark.timeout(10)
+@pytest.mark.timeout(20)
```
```suggestion
@pytest.mark.timeout(40)
```

Yarboa commented on tests/tests/tier0/proxy-service-fails-on-execstart/test_proxy_service_fails_on_execstart.py:
```diff
@@ -42,7 +42,7 @@ def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer
 
 @pytest.mark.skip(reason="Currently flaky. Tracked in https://github.com/containers/bluechi/issues/320. ")
-@pytest.mark.timeout(10)
+@pytest.mark.timeout(25)
```
```suggestion
@pytest.mark.timeout(40)
```

Yarboa commented on tests/tests/tier0/monitor-wildcard-unit-changes/test_monitor_wildcard_unit_changes.py:
```diff
@@ -31,7 +31,7 @@ def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer
 
 
-@pytest.mark.timeout(15)
+@pytest.mark.timeout(20)
```
```suggestion
@pytest.mark.timeout(40)
```

Yarboa commented on tests/tests/tier0/monitor-wildcard-node-reconnect/test_monitor_wildcard_node_reconnect.py:
```diff
@@ -20,7 +20,7 @@ def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer
 
 
-@pytest.mark.timeout(10)
+@pytest.mark.timeout(25)
```
```suggestion
@pytest.mark.timeout(40)
```

Yarboa reviewed (commented):
@dougsland 
Should that be sufficient?
